js/gdm/authPrompt.js

```diff
@@ -39,6 +41,14 @@ var BeginRequestType = {
     DONT_PROVIDE_USERNAME: 1
 };
 
+function _getMachineId() {
+    let machineId = getFileContents('/etc/machine-id') ||
+                    getFileContents('/var/lib/dbus/machine-id');
```

Comment:
There is no need to fall back to this path. `/etc/machine-id` is guaranteed to be created very early in boot on systemd-managed systems. `/var/lib/dbus/machine-id` is a symlink to it so if the former can't be read, falling back to this path won't help.

Reply:
Done.